server/src/main/java/io/crate/types/DataTypes.java

```diff
@@ -360,6 +366,8 @@ private static boolean safeConversionPossible(DataType type1, DataType type2) {
     }
 
     private static final Map<String, DataType> MAPPING_NAMES_TO_TYPES = Map.ofEntries(
+        entry(DataTypes.TIMETZ.getName(), DataTypes.TIMETZ),
```

Comment:
does this `time with time zone` mapping exist?

Reply:
Nope, with the question I realised, thank you:

https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html